test/cmdlineTests.sh

```diff
@@ -41,31 +43,33 @@ test "${output//[[:blank:]]/}" = "3"
 
 function compileFull()
 {
-    files="$*"
+    local files="$*"
+
     set +e
-    "$SOLC" --optimize \
-    --combined-json abi,asm,ast,bin,bin-runtime,clone-bin,compact-format,devdoc,hashes,interface,metadata,opcodes,srcmap,srcmap-runtime,userdoc \
-    $files >/dev/null 2>&1
-    failed=$?
+    local output=$( ("$SOLC" $FULLARGS $files) 2>&1 )
+    local failed=$?
     set -e
+
     if [ $failed -ne 0 ]
     then
         echo "Compilation failed on:"
-        cat $files
```

Comment:
You either have to give the name of the file that was compiled or the source code. If the file is a temporary file (i.e. extracted from tests or documentation), the contents should be shown.